modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -735,9 +735,22 @@ def pushDockerImageToGCR(imageName, tagName) {
 }
 
 /*
+ * Upload the artefact at artefact_path to the given s3_path. Uses the
+ * govuk-s3-artefact-creds for access.
+ */
+def uploadArtefactToS3(artefact_path, s3_path){
+  withCredentials([[$class: 'UsernamePasswordMultiBinding',
+                     credentialsId: 'govuk-s3-artefact-creds',
+                     usernameVariable: 'AWS_ACCESS_KEY_ID',
```

Comment:
Currently they're added to Jenkins manually and locked down to only be able to write to the artefact bucket, I didn't think we injected jenkins creds via puppet?